examples/WORKSPACE

```diff
@@ -1,24 +1,21 @@
 workspace(name = "examples")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
-http_archive(
+git_repository(
```

Comment:
This should maybe be changed to `http_archive` in the smoke test, as it's generally recommended to use `http_archive` over `git_repository` where possible, since it tends to be smaller, faster, and can utilize the repository cache.

Reply:
Updated